balancer/pickfirst/metrics_test.go

```diff
@@ -102,11 +102,31 @@ func (s) TestPickFirstMetrics(t *testing.T) {
 		t.Errorf("Unexpected data for metric %v, got: %v, want: %v", "grpc.lb.pick_first.disconnections", got, 0)
 	}
 
+	//Checking for subchannel metrics as well
```

Comment:
Nit: Space between the `//` and the start of the comment, and please terminate comment sentences with a period.